app/src/api/jobs.js

```diff
@@ -19,7 +19,19 @@ export const getAllJobsAndSkills = async () => {
   try {
     const res = await axios.get(`${JOB_ENDPOINT}/skills`);
     if (res) {
-      console.log(combineSkillsToJobs(res.data))
+      return combineSkillsToJobs(res.data);
+    }
+    throw new Error("No data returned from backend");
+  } catch (error) {
+    console.log(error);
+    return [];
+  }
+};
+
+export const getAllActiveJobsAndSkills = async () => {
```

Comment:
Do you think it might be a good idea to have one function getAllJobs... which takes in params to determine whether to get all, active, or inactive.  Is that good design? @theianchia @charmjj @branfats @deadline2359 maybe the others can chime in as well.

Reply:
I'm wondering if this should be in `jobSkills.js` instead. Having one function to perform this is fine, as even if we don't use some complex SQL join query to retrieve, it would still be this one function that is responsible for reusing and invoking the methods in `jobs.js` and `skills.js` to form this

Reply:
Should the function just call the BE endpoint straight, instead of reusing the methods in `jobs.js` and `skills.js`? If you reuse the functions then you'll be making multiple API calls instead of just 1.

Reply:
@quinncheong think ur original idea works, since that will "correspond" with the current state of the `/job/skills` endpoint in the backend

As for @theianchia 's suggestion on `jobSkills.js`, it might be a mental model mismatch since the backend houses this endpoint under the `/job` prefix. Unless we migrate the backend naming as well, the current frontend endpoint in `jobs.js` might be clearer